actions/structures.xml

```diff
@@ -28,18 +27,15 @@
                     <action id="test-authoring" name="Authoring" url="/taoTests/Tests/authoring" group="content" context="instance" binding="launchEditor">
                         <icon id="icon-edit"/>
                     </action>
-                    <action id="test-filter" name="Filter" context="*" group="search" binding="filter">
-                        <icon id="icon-filter"/>
-                    </action> 
-                    <action id="test-search" name="Search" url="/taoTests/Tests/search" context="*" group="search" binding="launchFinder">
-                        <icon id="icon-find"/>
-                    </action> 
                     <action id="test-class-new" name="New class" js="subClass" url="/taoTests/Tests/addSubClass" context="resource" group="tree">
                         <icon id="icon-folder-open"/>
                     </action>
                     <action id="test-delete" name="Delete" url="/taoTests/Tests/delete" context="resource" group="tree" binding="removeNode">
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="test-delete-all" name="Delete" url="/taoTests/Tests/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```

Comment:
The display name of 'delete all' is the same as for the 'delete one'.

Reply:
They aren't supposed to be displayed at the same time, multiple and single contexts are exclusives. The bug is fixed in tao-core